cockroach-admin/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "omicron-cockroach-admin"
+version = "0.1.0"
+edition = "2021"
+license = "MPL-2.0"
+
+[build-dependencies]
+omicron-rpaths.workspace = true
+
+[dependencies]
+anyhow.workspace = true
+camino.workspace = true
+chrono.workspace = true
+clap.workspace = true
+csv.workspace = true
+dropshot.workspace = true
+http.workspace = true
+illumos-utils.workspace = true
+omicron-common.workspace = true
+# See omicron-rpaths for more about the "pq-sys" dependency.
+pq-sys = "*"
```

Comment:
I'm a little surprised we need this + the build.rs script. I *thought* we would have needed this to get access to a copy of postgres - this is normally necessary for Diesel, but is it also necessary to invoke the CLI?

Reply:
I guess so - I didn't really look into it when I saw the rpath failure in CI.